src/bin/lib/write-errors-to-log.ts

```diff
@@ -0,0 +1,21 @@
+import * as fs from 'fs'
+import * as util from 'util'
+
+const appendFile = util.promisify(fs.appendFile)
+
+const writeErrorsToLog = (errors: Error | Error[], filename: string): Promise<void> => {
+  if (!Array.isArray(errors)) {
+    errors = [errors]
+  }
+
+  let json = errors.map((err) => {
+    const errObj = {}
+    Object.getOwnPropertyNames(err).forEach((key) => errObj[key] = err[key])
+    return JSON.stringify(errObj, null, 2)
```

Comment:
This formats the JSON in a nicely readable way for people, but that also means there are plenty of newlines.
I think for the error log file we should render one object per line, especially since that's how newline delimited JSON works ;)

Reply:
ah yes, i had human readability in mind but forgot about the newlines :woman_facepalming: